modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+
+[id="attaching-additional-labels-to-your-time-series-and-alerts_{context}"]
+= Attaching additional labels to your time series and alerts
+
+Using the external labels feature of Prometheus, you can attach additional custom labels to all time series and alerts leaving the Prometheus cluster.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
++
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Define a map of labels you want to add for every metric under `data/config.yaml`:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      externalLabels:
+        <key>: <value>
```

Comment:
Not critical, but I'd consider putting a <1> callout here and having the "Substitute..." sentence be the callout text.